soccer/gameplay/skills/pivot_kick.py

```diff
@@ -111,6 +111,7 @@ def remove_aim_behavior(self):
             self.remove_subbehavior('aim')
 
     def on_enter_capturing(self):
+        self.enable_shot_obstacle = False
```

Comment:
This is slightly vauge (why its being done).
Could you add a comment saying something like:
"If kick fails, disable shot obstacle for second try"